node-operators/run-aut/index.md

```diff
@@ -22,26 +22,81 @@ To connect to a network and sync, get the genesis and bootnode files if needed,
     mkdir autonity-chaindata
     ```
 
-1. Start autonity:
+2. Generate the `autonitykeys` private key file for your node. This must be performed on the host machine running the Autonity Go Client, using the `autonity genAutonityKeys --writeaddress` command:
```

Comment:
I think this step is fine but just to understand, why explicitly generate the autonitykeys instead of letting the client do it on its own?

Reply:
Good catch. This was written before `autinspect` command was available and generating was the only way to get the Public consensus key.

This can be removed.

Reply:
Removed in run binary and run docker sections - reverted to use default generation and keyfile location. See commit 992350d7e043c4ca203e3a6b05288bdf32900c5a.